feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go

```diff
@@ -250,22 +258,18 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) {
 }
 
 // createFlow returns a flow from atePort1 to the dstPfx, expected to arrive on ATE interface dsts.
-func createFlow(name string, ate *ondatra.ATEDevice, ateTop *ondatra.ATETopology, dsts ...*attrs.Attributes) *ondatra.Flow {
-	hdr := ondatra.NewIPv4Header()
-	hdr.WithSrcAddress(dutPort1.IPv4).
-		DstAddressRange().WithMin(dstPfxMin).WithMax(dstPfxMax).WithCount(dstPfxCount)
-
-	endpoints := []ondatra.Endpoint{}
-	for _, dst := range dsts {
-		endpoints = append(endpoints, ateTop.Interfaces()[dst.Name])
-	}
-
-	flow := ate.Traffic().NewFlow(name).
-		WithSrcEndpoints(ateTop.Interfaces()[atePort1.Name]).
-		WithDstEndpoints(endpoints...).
-		WithHeaders(ondatra.NewEthernetHeader(), hdr)
-
-	return flow
+func createFlow(t testing.TB, name string, ate *ondatra.ATEDevice, ateTop gosnappi.Config, dst *attrs.Attributes) string {
+	modName := strings.Replace(name, " ", "_", -1)
```

Comment:
it would be better to return an error here rather than replacing. just let the user know you can't have spaces in the key

Reply:
I removed the replacement. It's actually ok to have spaces in the otg flow names